conductor/__beta__/job/worker.py

```diff
@@ -29,15 +29,12 @@ def __init__(self, *args , **kwargs):
         self.job_title = "Deadline Worker"
         self.instance_count = 1
         
-        # Hard-coded to work with Maya for the time being
-        self.software_packages_ids = ["bcfe5df6e2361d77ca7d7b9da76e351b", "936dac0a489071942be623da35dd71fb"]
-        
-        self.cmd = "sudo -E /opt/Thinkbox/Deadline10/bin/launch_deadline.sh"
-        self.docker_image = "gcr.io/images-production/conductor_docker_base:deadline_debian"
+        self.cmd = "/opt/thinkbox/deadline/conductor/launch_deadline.sh" #CONDUCTOR_DEADLINE_LAUNCHER_PATH
         self.deadline_proxy_root = None
         self.deadline_ssl_certificate = None
-        self.deadline_use_ssl = True
-        self.deadline_client_version = "10.0.28.2" # "10.1.1.3"- this version is experiencing errors. Installation on docker is broken.
+        self.deadline_use_ssl = False

```

Comment:
Because it's more tricky to setup.. and more likely to fail out of the box. Do you think it could open us up to vulnerabilities if encryption isn't used? Should we force it?

Reply:
Basic security should be enabled by default, and users can make the call to disable it. This is a great opportunity for us to use our position to enforce/encourage our customers to follow security best practices, and to show that we care about the security of their IP.

Reply:
ok - I'll switch the default